utils/getSubscription.py

```diff
@@ -0,0 +1,82 @@
+from categories.models import Category,UserSubscription
+
+class getSubscription:
```

Comment:
Capital G here 😅

Reply:
Also have you made the corresponding changes in notifications and emails?

Reply:
Filename: `get_subscripions.py`?